e2e/cypress/tests/integration/files_and_attachments/disabled_file_upload_spec.js

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @files_and_attachments
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Upload Files - Settings', () => {
+    let channelUrl;
+
+    before(() => {
+        // # Go to system admin then verify admin console URL and header
+        cy.visit('/admin_console/site_config/file_sharing_downloads');
+        cy.url().should('include', '/admin_console/site_config/file_sharing_downloads');
+        cy.get('.admin-console__header', {timeout: TIMEOUTS.ONE_MIN}).
+            should('be.visible').
+            and('have.text', 'File Sharing and Downloads');
+
+        // # Set file sharing to false
+        cy.findByTestId('FileSettings.EnableFileAttachmentsfalse').click();
+        cy.get('#saveSetting').click();
+
+        // # Create new team and new user and visit test channel
+        cy.apiInitSetup({loginAfter: true}).then(({channelUrl: url}) => {
+            channelUrl = url;
+        });
+    });
+
+    beforeEach(() => {
+        cy.visit(channelUrl);
+    });
+
+    it('MM-T1149_1 Disallow file sharing in the channel', () => {
```

Comment:
Maybe a typo with correct value based on [MM-T1147](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T1147)?
```suggestion
    it('MM-T1147_1 Disallow file sharing in the channel', () => {
```